lib/stripe/core_resources/event.ex

```diff
@@ -67,7 +67,8 @@ defmodule Stripe.Event do
     :pending_webhooks,
     :request,
     :type,
-    :user_id
+    :user_id,
+    :account
```

Comment:
`Renames the user_id property on Connect-related event objects to account`

@erikreedstrom Saw this in the [changelog](https://stripe.com/docs/upgrades#api-changelog)

Reply:
Still need a link or is changelog enough?

Should `user_id` be removed or deprecated? Not sure what min supported version of API is.

Reply:
Perfect . So we can probably just remove the `user_id` since users haven't been able to rely on this field (nor have been for over a year).  Then we should be good to go!